emoji/src/test/java/com/vanniktech/emoji/EmojiManagerTest.java

```diff
@@ -49,10 +47,26 @@
     assertThat(EmojiManager.getInstance().getCategories()).isNotEmpty();
   }
 
+  @Test public void installEmptyCategory() {
+    final EmojiProvider emptyProvider = new EmojiProvider() {
+      @NonNull @Override public EmojiCategory[] getCategories() {
+        return new EmojiCategory[0];
+      }
+    };
+
+    assertThatThrownBy(new ThrowableAssert.ThrowingCallable() {
```

Comment:
I guess you meant an empty `EmojiCategory`? If not, could you go in detail what you mean?